src/handlers/select-answer.js

```diff
@@ -0,0 +1,17 @@
+import { setStatusClass } from '../views/set-status.js';
+/*import { displayQuestion } from "../views/questions-list.js"; */
+
+ export function selectAnswer(event){
+    const selectedButton = event.target;
+    let correct = selectedButton.dataset.correct;
```

Comment:
This is a very neat way to pass data from the element to the handler without having separate handlers for each button! Love it!

I'd store the answer _key_ here though (a, b, c, ...), and move the class name logic into this function. It'll also make @Veronika121's work to hook it up simpler.